Jenkinsfile

```diff
@@ -59,41 +59,44 @@ pipeline {
       }
       post { failure { slackSend color: '#cb2431' , channel: '#kevm' , message: "Packaging Phase Failed: ${env.BUILD_URL}" } }
       stages {
-        stage('Ubuntu Bionic') {
+        stage('Ubuntu Focal') {
           stages {
-            stage('Build Ubuntu Bionic') {
+            stage('Build Ubuntu Focal') {
               agent {
                 dockerfile {
                   additionalBuildArgs '--build-arg K_COMMIT="${K_VERSION}" --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
                   reuseNode true
                 }
               }
               steps {
-                dir('bionic-build') {
+                dir('focal-build') {
                   checkout scm
-                  sh './package/debian/package bionic'
+                  sh './package/debian/package focal'
                 }
-                stash name: 'bionic', includes: "kevm_${env.VERSION}_amd64.deb"
+                stash name: 'focal', includes: "kevm_${env.VERSION}_amd64.deb"
               }
             }
-            stage('Test Ubuntu Bionic') {
+            stage('Test Ubuntu Focal') {
               agent {
                 dockerfile {
                   filename 'package/debian/Dockerfile.test'
-                  additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) --build-arg BASE_IMAGE=ubuntu:bionic'
+                  additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) --build-arg BASE_IMAGE=ubuntu:focal'
                   reuseNode true
                 }
               }
-              options { skipDefaultCheckout() }
+              options {
+                skipDefaultCheckout()
+                timeout(time: 25, unit: 'MINUTES')
+              }
               steps {
-                dir('bionic-test') {
+                dir('focal-test') {
                   checkout scm
-                  unstash 'bionic'
+                  unstash 'focal'
                   sh '''
                     export KLAB_OUT=$(pwd)
-                    sudo apt-get update
-                    sudo apt-get upgrade --yes
-                    sudo apt-get install --yes ./kevm_${VERSION}_amd64.deb
+                    sudo DEBIAN_FRONTEND=noninteractive apt-get update
+                    sudo DEBIAN_FRONTEND=noninteractive apt-get upgrade --yes
+                    sudo DEBIAN_FRONTEND=noninteractive apt-get install --yes ./kevm_${VERSION}_amd64.deb
```

Comment:
```suggestion
                    export DEBIAN_FRONTEND=noninteractive
                    sudo apt-get update
                    sudo apt-get upgrade --yes
                    sudo apt-get install --yes ./kevm_${VERSION}_amd64.deb
```

Reply:
No, this doesn't work, `sudo` is not carrying through environment variables properly.